src/main/java/se/kth/sda/skeleton/posts/Post.java

```diff
@@ -13,6 +18,8 @@ public Post(String body) {
         this.body = body;
     }
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
should these annotated be on the id field rather than the getter?

Reply:
Placed above the field id